Osiris/Hacks/Visuals.cpp

```diff
@@ -393,3 +399,25 @@ void Visuals::skybox() noexcept
         memory->loadSky(sv_skyname->string);
     }
 }
+
+std::wstring towstring(std::string str)
+{
+    return strconverter.from_bytes(str);
+}
+
+void Visuals::showVelocity() noexcept
```

Comment:
Why do it in visuals? Misc would be the place too put this

Reply:
It renders text on the screen so i believe it belongs in visuals